tests/shared/src/test/scala/cats/tests/SyntaxSerializationSuite.scala

```diff
@@ -36,7 +37,7 @@ class SyntaxSerializationSuite extends CatsSuite {
   )
 
   checkAll("SemigroupalOps[Option, Int]",
-           SerializableTests.serializable(cats.syntax.all.catsSyntaxSemigroupal[Option, Int](None))
+           SerializableTests.serializable(cats.syntax.all.catsSyntaxSemigroupal[Option, Int](None)(Semigroupal[Option]))
```

Comment:
Hmm, what changed here? Is there now some kind of ambiguous implicit problem?

Reply:
yeah, `catsSyntaxSemigroupal` implicit method was used directory in this test, and since we've removed implicit params I guess we need to pass `Semigroupal` instance explicitly.

Btw, should syntax classes be serializable? This test was introduced in https://github.com/typelevel/cats/pull/3439, and seems to required syntax classes to be serializable🤔